tests/config/rsc/multi-line-config.yaml

```diff
@@ -0,0 +1,62 @@
+levels:
+  example:
+    reporting:
+      do_nothing:
+    tool:
+      catkin_lint:
+        flags: >
+          "-W2
+          --ignore
+          DESCRIPTION_BOILERPLATE,
+          DESCRIPTION_MEANINGLESS,
+          GLOBAL_VAR_COLLISION,
+          LINK_DIRECTORY,
+          LITERAL_PROJECT_NAME,
+          TARGET_NAME_COLLISION"
+      cpplint:
+        flags: >
```

Comment:
Since we're claiming support for multi-line yaml, I think this file should use multiple different methods of multi-line breaks to make sure we test as many as we know about.
Alternatively, we should change the claim to just the method we know works and is tested.